chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java

```diff
@@ -1296,7 +1296,11 @@ private void createInitialTab() {
                     "MobileStartup.LoadedHomepageOnColdStart", startupHomepageIsNtp);
         }
 
-        getTabCreator(false).launchUrl(url, TabLaunchType.FROM_CHROME_UI);
+        if (CommandLine.getInstance().hasSwitch(BaseSwitches.ENABLE_SERVICE_OFFLOADING)) {
+            getTabCreator(true).launchUrl(url, TabLaunchType.FROM_CHROME_UI);
+        } else {
+            getTabCreator(false).launchUrl(url, TabLaunchType.FROM_CHROME_UI);
+        }
```

Comment:
To minimize code changes, how about attaching --disk-cache-size=1 or --incognito as a runtime option using meerkat?

Reply:
I hope runtime flags work with meerkat. When I try it on the command line, it didn't work.

Reply:
And the incognito mode is always false in the initial tab regardless of the --incognito
> getTabCreator(false)

Reply:
@YoungsooChoi Can you check if --disk-cache-size=1 works?

Reply:
@Chikdolman 
I have tested with '--disk-cache-size=' but it's not working.

2020-11-24 15:21:41.621 16843-16843/com.samsung.android.castanets E/cr_ActivitiyDispatcher: YSYS ::: args = com.samsung.android.castanets --disk-cache-size=1 --enable-service-offloading --loadurl=file:///sdcard/Download/index.html

Reply:
Checked that --disk-cache-size did not affect.